package/mediacenter-osmc/patches/all-104-support-for-hardware-keyboards.patch

```diff
@@ -219,7 +228,7 @@ index ef0d494..e46d9bb
 +          CLog::Log(LOGDEBUG, "CLinuxInputDevice::KeyEvent: using 0x%04x as unicode", keyMapValue);
 +        }
 +        // use the extended character set but only if an 'ascii' key was pressed
-+        else if (keyMapValue < 256 && keyMapValue > 0x20 && devt.key.keysym.sym < 128)
++        else if (keyMapValue < 256 && keyMapValue > 0x1f && devt.key.keysym.sym < 128)
```

Comment:
@graham8 Why you've changed this only in one place? What about here
https://github.com/osmc/osmc/pull/395/files#diff-073bf2b78ad1a74900a6e27f29f5da9bR244

Reply:
Unnecessary.  Out of a keyboard dialog, kodi picks up the key directly and uses it as pause.  I haven't noticed any problems but will change it if you find otherwise.